src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let finalPrice = 0;
+
+  return 'hello serhii';
+
+  if (days >= 7) {
```

Comment:
Consider returning the calculated value immediately within the `if` blocks to simplify the function and avoid unnecessary `else` statements.